src/stores/inactive-users.ts

```diff
@@ -0,0 +1,49 @@
+import { sleep } from './../services/utils'
```

Comment:
Do we need stores for users at all?

Imo this data would be fetched from backend and is useful only for that page (it doesn't have to be reactive).

While for profile/pricing plan/payment method - that data is global and it makes sense to keep it universally available.

Reply:
Let's keep store if rework takes a lot though.